targets/TARGET_NXP/scripts/mbed_set_post_build_nxp.cmake

```diff
@@ -11,7 +11,7 @@ function(mbed_post_build_lpc_patch_vtable mbed_target_name)
 
     set(post_build_command
         COMMAND ${Python3_EXECUTABLE} ${MBED_PATH}/targets/TARGET_NXP/scripts/LPC.py
-        ${CMAKE_BINARY_DIR}/${APP_TARGET}.bin
+        ${CMAKE_BINARY_DIR}
```

Comment:
this should be current binary dir ?

Reply:
In this instance it has to be `CMAKE_BINARY_DIR`.
Because with Greentea tests,  `CMAKE_CURRENT_BINARY_DIR` here is not where we want.
For example when building the hal qspi Greentea test, `CMAKE_CURRENT_BINARY_DIR` is `/path/to/mbed-os/hal/tests/TESTS/mbed_hal/qspi/cmake_build/LPC546XX/develop/ARM/build/targets/TARGET_NXP/TARGET_MCUXpresso_MCUS/TARGET_MCU_LPC546XX/`. Whilst `CMAKE_BINARY_DIR` is `/path/to/mbed-os/hal/tests/TESTS/mbed_hal/qspi/cmake_build/LPC546XX/develop/ARM/`.